contracts/eoslib/mainpage.md

```diff
@@ -1,19 +1,38 @@
-Welcome to the eos.io Documentation
+Welcome to the EOS.io Documentation
 -----------------------------------
 
-@note this documentation is under development and may be incorrect. 
-
-This documentation describes the APIs available to eos.io application developers. This is the best place to
-start your journey into smart contract development.
-
-## Environment 
-
-Before you can start building your applications you will need to properly configure your development 
-environment. Currently you can only develop EOS.IO applications on Mac OS X or Linux.
-
-## Examples
-
-
-## Reference
-
-@ref database
+@note This documentation is in progress and subject to change due present rapid development. Please report inaccuracies identified to the [EOS developer telegram group](https://t.me/joinchat/EaEnSUPktgfoI-XPfMYtcQ)
+
+Below is your gateway to everything EOS.
+
+## EOS
+ - [EOS.IO Website](https://www.eos.io/)
+ - [Blog](https://steemit.com/@eosio)
+ - [Roadmap](https://github.com/EOSIO/Documentation/blob/master/Roadmap.md/)
+ - [Telegram](https://www.eos.io/chat/)
+ - [White Paper](https://github.com/EOSIO/Documentation/blob/master/TechnicalWhitePaper.md/)
+
+## Wallets
```

Comment:
This separation goes against the spec I was given which was to separate based on *User Case* not functionality.